dockerfiles/mpi-worker-start.sh

```diff
@@ -47,6 +48,20 @@ for gid in `id -G`; do
   fi
 done
 
+cat << EOF > "$DOMINO_HOME_DIR/.profile"
+if [ "\$BASH" ] && [ -f ~/.bashrc ]; then
+  . ~/.bashrc
+fi
+EOF
+chmod 644 "$DOMINO_HOME_DIR/.profile"
+
+rm -f "$DOMINO_HOME_DIR/.bashrc"
+touch "$DOMINO_HOME_DIR/.bashrc"
+printenv | grep PATH | sed 's;^;export ;' >> "$DOMINO_HOME_DIR/.bashrc"
```

Comment:
I take it that it is a conscious decision to only copy certain vars? We are intentionally not copying `ENV` vars from the image? Or do those already work?

Reply:
Yes, we copying only relevant variables. The source of truth is the current **runtime environment** (which is ENV + whatever bash init files got sourced).